releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -0,0 +1,38 @@
+# azure-spring-cloud-appconfiguration-config
+
+[Source code][source_code] | [Package (Maven)][package] | [README][readme] | [Product documentation][docs] | [Samples][samples]
+
+# azure-spring-cloud-appconfiguration-config-web
+
+[Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 2.0.0-beta.1 - May 5, 2021
+
+* Enables loading of configurations from multiple config stores.
+* Each store can load configurations by using key/label `selects`. Many `selects` can be used, default is key = `/applicaiton/*` with label = `${spring.profiles.active}` or if null `\0`. See [README][readme] for full configuration.
+* Authentication vis Connection String, Managed Identity, or any method supported by Azure Identity SDK, see Token Credential Provider in README.
+* Each individual store can have monitoring enabled and disabled and have a cache-expiration time set.
+  * Monitoring requires at least one trigger to be set. `spring.cloud.azure.appconfiguration.stores[0].monitoring.triggers[0].key` is required `spring.cloud.azure.appconfiguration.stores[0].monitoring.triggers[0].label` is optional.
+  * Monitoring has both Pull and Push model options.
+    * Pull checks the config store for changes based on Activity when the cache has expired.
+    * Push involves setting up a Azure Event Grid Web Hook to notify the client application that a change has occurred. Two Spring Actuator Endpoints exist to connect to. `appconfiguration-refresh` triggers the cache to reset on configurations on an application. `appconfiguration-refresh-bus` triggers a refresh on all instances subscribed to the same Service Bus.
```

Comment:
This is not enough information for a user to figure out how to use the Push model. We need a full document for this. In the release notes, I would suggest being descriptive only and leave the technical details in the doc.

BTW, I would avoid using the term "client application". I can cause confusion.

Reply:
@zhenlan We have a Push Model doc, it is in PR waiting for review.

Do you have a preferred word besides client application? I am trying to be clear where the web-hook connects to.